docs/reference/commandline/run.md

```diff
@@ -17,6 +17,7 @@ parent = "smn_cli"
       -a, --attach=[]               Attach to STDIN, STDOUT or STDERR
       --add-host=[]                 Add a custom host-to-IP mapping (host:ip)
       --blkio-weight=0              Block IO weight (relative weight)
+      --blkio-weight-device="" 	    Block IO weight (relative weight for device)
```

Comment:
minor nit, and not in docs review yet, but you have a "tab" in here, causing it to align improperly